main.go

```diff
@@ -70,10 +113,10 @@ func main() {
 		},
 	}
 
-	whip.Publish(stream, mediaEngine, iceServers, false)
+	whip.Publish(stream, mediaEngine, iceServers, true)
 
 	fmt.Println("Press 'Enter' to finish...")
 	bufio.NewReader(os.Stdin).ReadBytes('\n')
 
-	whip.Close()
+	whip.Close(true)
```

Comment:
i thikn we should avoid having to pass the parameter in each method.  Will create a PR to move it to the WHIPClient object